### ISSUE TITLE
[BUG] Null object reference Android

### ISSUE THREAD

**Issue by Nico3652** [nizwar/screen_capture_event#2]
Thanks for the plugin. 

I get a null pointer event on Android on screen recording : 

```
E/AndroidRuntime(14778): java.lang.NullPointerException: Attempt to invoke virtual method ‘boolean java.lang.String.contains(java.lang.CharSequence)’ on a null object reference
E/AndroidRuntime(14778): 	at id.nizwar.screen_capture_event.ScreenCaptureEventPlugin$1.lambda$onEvent$0$ScreenCaptureEventPlugin$1(ScreenCaptureEventPlugin.java:81)
E/AndroidRuntime(14778): 	at id.nizwar.screen_capture_event.ScreenCaptureEventPlugin$1$$ExternalSyntheticLambda0.run(Unknown Source:6)
E/AndroidRuntime(14778): 	at android.os.Handler.handleCallback(Handler.java:899)
E/AndroidRuntime(14778): 	at android.os.Handler.dispatchMessage(Handler.java:100)
E/AndroidRuntime(14778): 	at android.os.Looper.loop(Looper.java:238)
E/AndroidRuntime(14778): 	at android.app.ActivityThread.main(ActivityThread.java:7884)
E/AndroidRuntime(14778): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(14778): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:512)
E/AndroidRuntime(14778): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:998)
```

**Comment by nizwar** [comment]
Hello, thankyou for point out the bug, it will be fixed on next release..

**Comment by kvenn** [comment]
@nizwar thanks for the lib. 

Has this been fixed? I'm still seeing this issue for Android devices on version 1.0.3.

**Comment by nizwar** [comment]
Fixed on 1.1.0
